static/js/custom.js

```
@@ -27,58 +21,40 @@
-  // We assume Docusaurus updates the query string automatically when a tab is clicked.
-  // Here we simply wait a bit and then check if the URL still has a hash.
-  // If not, we restore it from currentHash.
-  document.addEventListener('DOMContentLoaded', function() {
-    // Adjust the selector below if necessary.
-    document.querySelectorAll('.tab').forEach(function(tab) {
-      tab.addEventListener('click', function(e) {
-        // Allow Docusaurus to update the URL.
-        setTimeout(function() {
-          // If the URL hash is missing, restore it from currentHash.
-          if (!window.location.hash && currentHash) {
+          if (h3 && h3.id && h3.id !== window.location.hash.slice(1)) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)